src/Microsoft.Management.Configuration.Processor/Microsoft.Management.Configuration.Processor.csproj

```diff
@@ -52,14 +53,53 @@
     <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(WinGetCsWinRTEmbedded)'!='true'">
     <ProjectReference Include="..\Microsoft.Management.Configuration.Projection\Microsoft.Management.Configuration.Projection.csproj">
       <OutputItemType>Content</OutputItemType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <ReferenceOutputAssembly>True</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(WinGetCsWinRTEmbedded)'=='true'">
+    <ProjectReference Include="..\Microsoft.Management.Configuration\Microsoft.Management.Configuration.vcxproj">
+      <OutputItemType>Content</OutputItemType>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </ProjectReference>
+  </ItemGroup>
+  
+  <PropertyGroup Condition="'$(WinGetCsWinRTEmbedded)'=='true'">
+    <DefineConstants>$(DefineConstants);WinGetCsWinRTEmbedded</DefineConstants>
+    <CsWinRTComponent>false</CsWinRTComponent>
+    <CsWinRTIncludes>
+      Microsoft.Management.Configuration;
+      Windows.ApplicationModel;
+      Windows.Data;
+      Windows.Devices;
+      Windows.Foundation;
+      Windows.Gaming;

```

Comment:
Out of curiosity - why do we need `Windows.Gaming`?

Reply:
It appears to be in the closure of things we need.  The list was generated by iteratively building and adding namespaces until the unrecognized namespace/type errors stopped.  It is possible that a more targeted set of types could be determined to prevent quite so much from being generated.

I will give that a try to see if it isn't too painful, as it will reduce build time and potentially allow us to not need to set up trimming.

Reply:
Nope, very painful.

I did however try removing just `Windows.Gaming` and found that it was `Windows.UI.Input.Preview.Injection` that was pulling it in.  Since we can selectively exclude things, this pruning seems like a more sane way to get rid of some of the excess.

Reply:
Well, 25% reduction isn't too bad for just trimming off one small portion. Thanks!!

Reply:
Does the same optimization apply to Engine, or is that one more painful?